### ISSUE TITLE
Translate Telephone is dead

### ISSUE THREAD

**Issue by Billy-man** [kmh11/kmh.zone#1]
Not sure if you care much about it, but the Translate telephone went dead. shows me a black screen saying there's no web page when I try to access it. I actually liked using it, so I hope you can get it working again.

**Comment by kmh11** [comment]
Yeah, Google started cracking down on automated access to Google Translate. I used Tor to change IP for a while, but then at one point it just stopped working for reasons I haven't investigated. Now that I know other people use it, I may look into it soon and see if I can get it back up.

**Comment by Billy-man** [comment]
Ah, good to hear you'll try to fix it. It's a very nifty tool, so all I can do is to wish for a speedy fix of it.
